Andriy_Povzun/HW8_Andriy_Povzun/Task4.py

```diff
@@ -0,0 +1,55 @@
+def my_max(objects: list, amount_of_result=1):
+    max_number = objects[0]
+    second_max = objects[0]
+    if amount_of_result > 2 or amount_of_result < 1:
+        raise Exception("The maximum number of objects that can be returned is 2")
```

Comment:
why so? I want more max numbers to be printed not 1 or 2 ))

Reply:
Okay, I'm fix it with your support, thanks)